apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1094,6 +1082,7 @@ namespace MWMechanics
         }
 
         // Make surrounding actors within alarm distance respond to the crime
+        // TODO: Vanilla doesn't make neighbors attack player if one of his follower/summon attacks first (probably vanilla bug)
```

Comment:
This seems to be because in vanilla followers only give you a crime for a kill, not for attacking. The same thing will happen if you kill an NPC instantly. You will only get the murder bounty and nearby NPCs won't attack you. I guess the code to have them respond is tied to the "on attack" code.